src/main/resources/reference.conf

```diff
@@ -156,4 +156,46 @@ cassandra-snapshot-store {
       parallelism-max = 8
     }
   }
+}
+
+cassandra-query-journal {
+  # Implementation class of the Cassandra ReadJournalProvider
+  class = "akka.persistence.cassandra.query.journal.CassandraReadJournalProvider"
+
+  # Stream queries database in defined `refresh-interval`.
+  refresh-interval = 3s
+
+  # How many events to fetch in one query (replay) and keep buffered until they
+  # are delivered downstreams.
+  max-buffer-size = 100
+
+  # Target partition size of index tables. Must match cassandra-journal configuration.
+  target-partition-size = 500000
```

Comment:
I think we should use a reference to the write plugin config instead, like https://github.com/krasserm/akka-persistence-cassandra/pull/128/files#diff-fa2e842b741241072d2436e8ae36c3d2R170

It must be possible to use many write plugins and many query plugins in same actor system, but we can say that one query plugin must be "connected" to a write plugin.